--- konsole-17.07.90/src/ColorScheme.cpp.0000~	2017-08-07 20:50:22.753207167 +0200
+++ konsole-17.07.90/src/ColorScheme.cpp	2017-08-07 20:51:05.240134479 +0200
@@ -485,7 +485,7 @@ bool ColorSchemeWallpaper::draw(QPainter
         return false;
     }
 
-    if (qFuzzyCompare(1.0, opacity)) {
+    if (qFuzzyCompare(qreal(1.0), opacity)) {
         painter.drawTiledPixmap(rect, *_picture, rect.topLeft());
         return true;
     }
